Assets/Scripts/Managers/HouseManager.cs

```diff
@@ -18,13 +18,18 @@ public enum HouseMode { Explore, Decorate }
 	private List<HouseItem> houseItems;
 	private float houseValue = 0;
 
+	private AudioManager audioManager;
+
 	private void Start()
 	{
 		SpawnSerializedPlaceables();
 		houseValue = CalculateHouseRating(houseItems); // assign total value here
 
         Debug.Log("HouseRating: "+houseValue);
 		SetHouseMode(HouseMode.Explore);
+
+		audioManager = FindObjectOfType<AudioManager>();
+		audioManager.Play("Building");
```

Comment:
These lines would instead be 
```suggestion
		AudioManager.Instance.Play("Building");
```